src/Engine/ProtoCore/Core.cs

```diff
@@ -828,7 +828,7 @@ public void ResetSSASubscript(Guid guid, int subscript)
 
         public void Cleanup()
         {
-            CLRModuleType.ClearTypes();
+            //CLRModuleType.ClearTypes();
```

Comment:
When a new workspace is opened, this static list of types was cleared which caused class redefinition errors downstream as some of the same types were being loaded again due to this list being empty.